ci/build-unix.yml

```diff
@@ -92,6 +92,16 @@ steps:
       bazel build //ledger-service/http-json:http-json-binary_deploy.jar
       cp bazel-bin/ledger-service/http-json/http-json-binary_deploy.jar $(Build.StagingDirectory)/$JSON_API
       echo "##vso[task.setvariable variable=json-api;isOutput=true]$JSON_API"
+
+      TRIGGER=trigger-runner-${{parameters.release_tag}}.jar
```

Comment:
Similar question here, how about `daml-trigger-*.jar` or `daml-trigger-runner-*.jar` if we want to reserve the former for the trigger service.

Reply:
I have a slight preference for keeping our options open with the service, as that is the long-term one, but ultimately your call. I'll change to `daml-trigger-runner` for now.

Reply:
yeah I also prefer the `daml-trigger-runner` option, thanks!